### ISSUE TITLE
Permission to "steal" your great work

### ISSUE THREAD

**Issue by teharris1** [pfrommerd/insteon-terminal#15]
I am very impressed with your work here and am developing a python module for Insteon controls. It is primarily used for Home Assistant but is designed to be Insteon driven, not HA driven. Here is the git repo: [python-insteonplm](https://github.com/nugget/python-insteonplm). I am currently working on a set of All-Linking utilities to add to the module and would like to use some of your ideas here. Our design is not that different but not fully transferable either of course. 

Bottom line is I will be creating this functionality and if I use your work want to give you credit but also make sure you are OK with it. I know technically I don't need to do that since it is in the public domain and I will be recoding from scratch anyway but want to be respectful and show support for your work. I know how hard this stuff is.

BTW, I have seen some of your work in OpenHAB too. Very good stuff. Happy to collaborate on different topics.

**Comment by pfrommerd** [comment]
As you noted, the code base has been put entirely into the public domain, so feel free to use the code for whatever you like. Any tools which make working with Insteon devices easier is good for the whole Insteon/Home Automation community. I've considered porting the insteon terminal into pure python to remove the last vestiges of Java code, but mainly for time reasons haven't, and so am glad that you are working on a dedicated python library for plm support.
